PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -339,6 +354,7 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
     ),
 )
 
+
 (run2_egamma).toModify(
```

Comment:
Can you do the same thing below as the cutBased ID for Run2?

Reply:
@swertz, thanks for looking into the missing part. I did the relevant changes for not storing the Run3 IDs in Run2. The pull request is updated.